Updated Siren Markets tvl to include polygon and mainnet

MysticDakra opened the pull request:
Update the tvl so we are calculating the correct value using mainnet and polygon

##### Twitter Link: 
- https://twitter.com/sirenprotocol?s=20


##### List of audit links if any:


##### Website Link:
- https://sirenmarkets.com/
- https://app.sirenmarkets.com/trade


##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
![white_siren_circle](https://user-images.githubusercontent.com/83468031/126713668-341dcdc4-b10c-4aa2-b43f-b78284a5eeac.png)



##### Current TVL: 
- Need to include this




##### Chain:  ETH


##### Coingecko ID (so your TVL can appear on Coingecko):  
- https://www.coingecko.com/en/coins/siren


##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
- Already listed 


##### Description/bio (to be shown on DefiLlama): 
- Decentralized option markets on Ethereum.


##### Token address and ticker if any:
- Ticker: SI
- Address: 0xD23Ac27148aF6A2f339BD82D0e3CFF380b5093de


##### Category (Yield/Dexes/Lending/Minting): 
- Options

##### [Optional] ETH addresss (to receive a LlamaPunk):
- My address: 0x04BD82Bf46de00a048d66E9Eb00C5Bc28A07Cfb5

zareth-san commented on projects/siren/index.js:
```diff
@@ -3,61 +3,115 @@
+
+  const POLYGON_GRAPH_URL = 'https://api.thegraph.com/subgraphs/name/sirenmarkets/protocol-v2-matic'
+
+  const POLYGON_SERIES_ADDRESS = '0x716c543b39a85aac0240ba7ed07e79f06e1fed48'
```
First, see my comment down below about using SeriesVault, rather than SeriesController

You should fetch this from the POLYGON_GRAPH_URL, that way we can de-couple this adapter from our protocol. So, if we ever change the address used for the SeriesVault, we won't need to remember to update this address.

```
query {
  seriesVaults {
    id
  }

}

```

MysticDakra replied:
Right right that makes sense ty for review ill make these edits

zareth-san commented on projects/siren/index.js:
```diff
@@ -3,61 +3,115 @@
+    // Get collateral in Series
+    for (let im = 0; im < amm.series.length; im++) {
+
+      const response = await sdk.api.erc20.balanceOf({
```
this is going to get the collateral token balance on the SeriesController, but recall we don't store any tokens on the SeriesController. I think what you want here is the SeriesVault, which is what stores the collateral locked in the protocol used to cover the options.

zareth-san commented on projects/siren/index.js:
```diff
@@ -69,6 +123,25 @@
+    collateralToken {
+      id
+    }
+    series{
```
See my comment above, we don't need the series anymore, just the series vault

0xngmi commented:
Saw that this PR just got reviewed, please ping me when you want me to review and merge

MysticDakra commented:
Updated this I removed the loop because we shouldnt have more than one series vault at a time right?  
If we expect to have more than one vault I can make this to loop over what is returned from the series vault
